admin/composer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
+        "silverstripe/blog": "dev-master"
```

Comment:
Needs to be removed, we're not requiring blog in core. I assume that's a dev artefact.